dev/test-studio/structure/resolveStructure.ts

```diff
@@ -33,6 +33,7 @@ import {delayValue} from './_helpers'
 import {typesInOptionGroup} from './groupByOption'
 
 export const structure: StructureResolver = (S, {schema, documentStore, i18n}) => {
+  i18n.loadNamespaces(['testStudio']) // @todo: confirm – is this idiomatic usage?
```

Comment:
Do you know if this necessary here? 
cc: @bjoerge @rexxars

Reply:
If it's been added in this branch I would assume it's not, and maybe introduced by a merge error?